product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter.mdx

```diff
@@ -4,41 +4,44 @@ title: "Installing the MySQL Foreign Data Wrapper"
 
 <div id="installing_the_mysql_data_adapter" class="registered_link"></div>
 
-The MySQL Foreign Data Wrapper can be installed with an RPM package. During the installation process, the installer will satisfy software prerequisites.
+
 
 <div id="using_an_rpm_package_to_install_the_data_adapter" class="registered_link"></div>
 
-## Installing the MySQL Foreign Data Wrapper using an RPM Package
+## Installing on Linux x86-64 (amd64)  
 
-You can install the MySQL Foreign Data Wrapper using an RPM package on the following platforms:
+You can install the MySQL Foreign Data Wrapper on the following operating systems:
 
--   [RHEL 7](#rhel7)
--   [RHEL 7 PPCLE](#centos7_PPCLE)
 -   [RHEL 8](#rhel8)
+-   [RHEL 7](#rhel7)
+-   [Rocky Linux/AlmaLinux 8](#centos8)
 -   [CentOS 7](#centos7)
--   [CentOS 8](#centos8)
+-   [Ubuntu/Debian](#on-debian-or-ubuntu)
 
-<div id="rhel7" class="registered_link"></div>
 
-### On RHEL 7
+
+<div id="rhel8" class="registered_link"></div>
```

Comment:
I assume this is just to give an anchor for linking?

Reply:
yep